src/main/java/github/CollaboratorServiceEx.java

```diff
@@ -0,0 +1,50 @@
+package github;
+
+import org.apache.logging.log4j.Logger;
+import org.eclipse.egit.github.core.IRepositoryIdProvider;
+import org.eclipse.egit.github.core.User;
+import org.eclipse.egit.github.core.client.GitHubClient;
+import org.eclipse.egit.github.core.service.CollaboratorService;
+import org.eclipse.egit.github.core.service.UserService;
+import util.HTLog;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CollaboratorServiceEx extends CollaboratorService {
+
+    private static final Logger logger = HTLog.get(PullRequestServiceEx.class);
+
+    private final UserService userService = new UserService();
+
+    public CollaboratorServiceEx(GitHubClient client) {
+        super(client);
+    }
+
+    /**
+     * Gets the list of collaborators using CollaboratorService
+     * Then iterate through the list to get each collaborator using USerService
+     * This is done because the GitHub response for GET /repos/:owner/:repo/collaborators
+     * does not return a User with all the attributes of the user
+     *
+     * @param repository
+     * @return list of collaborators
+     * @throws IOException
+     */
+    @Override
+    public List<User> getCollaborators(IRepositoryIdProvider repository) throws IOException {
+        List<User> collaborators = super.getCollaborators(repository);
+        List<User> collaboratorsWithFullDetails = new ArrayList<>();
+
+        collaborators.forEach(user -> {
```

Comment:
This is a mapping operation, so you can `.stream().map()` over `collaborators` to get the new list.